packages/avida_test_utils/src/sdjwt/verifier_tests.rs

```diff
@@ -135,11 +98,442 @@ fn basic() {
     let resp = contract
         .verify(
             Binary::from(presentation.as_bytes()),
-            fx_route_id,
-            Some(caller_app.to_string()),
+            FX_ROUTE_ID,
+            Some(FIRST_CALLER_APP_ADDR.to_string()),
         )
-        .call(caller_app)
+        .call(FIRST_CALLER_APP_ADDR)
         .unwrap();
 
     println!("resp: {:?}", resp);
```

Comment:
This was incomplete by me, for this PR,  would be good before we merge to test verification and test `MAX_PRESENTATION_LEN` as well.

Reply:
Added missing `PresentationTooLarge` related check here https://github.com/nymlab/AVIDA/pull/1/commits/4e4cac9cb2b34a5b17d9b7f622db5089fd33bd60, will notify you after i add full unit test coverage for the `verify` functionality

Reply:
ah! sorry seems previous comment was not clear. 

Instead of  `println!("resp: {:?}", resp);`

We should decode the `resp.data` Binary and check that is actually returns `validate` result (true or false) in the `resp` struct which is something like `AppResponse { events: [Event { ty: "execute", attributes: [Attribute { key: "_contract_address", value: "contract0" }] }], data: Some(Binary(74727565)) }`